include/functions.h

```diff
@@ -990,8 +990,8 @@ void Color_RGBA8_Copy(Color_RGBA8* dst, Color_RGBA8* src);
 void func_80078884(u16 sfxId);
 void func_800788CC(u16 sfxId);
 void func_80078914(Vec3f* arg0, u16 sfxId);
-void Health_InitData(GlobalContext* globalCtx);
-void Health_UpdateData(GlobalContext* globalCtx);
+void HealthMeter_Init(GlobalContext* globalCtx);
+void HealthMeter_Update(GlobalContext* globalCtx);
 void Health_Draw(GlobalContext* globalCtx);
 void Health_HandleCriticalAlarm(GlobalContext* globalCtx);
 u32 Health_IsCritical(void);
```

Comment:
Might as well rename the whole system to HealthMeter_ for consistency, if we think that's better (at the very least for the Draw function)